scripts/update/jellyfin.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+#
+if [[ -f /install/.jellyfin.lock ]]; then
+    # source the functions we need for this script.
+    . /etc/swizzin/sources/functions/utils
+    # Get our main user credentials using a util function.
+    username="$(_get_master_username)"
+    dist_info # get our distribution ID, set to DIST_ID, and VERSION_CODENAME, set to DIST_CODENAME, from /etc/os-release
```

Comment:
`scripts/update/jellyfin.sh: line 8: dist_info: command not found`
Need to make sure this is sourced in